src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Initialization.cs

```diff
@@ -19,6 +19,14 @@ static Crypto()
         }
     }
 
+    internal static partial class OpenSsl
```

Comment:
It is not clear to my why the CryptoInitializer is not sufficient and this new initialization invocation is needed.

Reply:
`CryptoInitializer` is still used to initialize. What was insufficient is to have only `Interop.Crypto` to trigger the initialization. `Interop.OpenSsl` is another surface area that should trigger initialization as there are no guarantees that `Interop.Crypto` methods are always the first to be used.

One fairly common way to hit uninitialized shim was via `Interop.OpenSsl.OpenSslVersionNumber`, for example.